Doc/library/datetime.rst

```diff
@@ -436,6 +436,21 @@ Other constructors, all class methods:
    d``.
 
 
+.. classmethod:: date.fromisoformat(date_string)
+
+  Return a :class:`date` corresponding to a *date_string* in one of the ISO 8601
+  formats emitted by :meth:`date.isoformat`. Specifically, this function supports
+  strings in the format(s) ``YYYY-MM-DD``.
```

Comment:
There is only one format emitted by _date.isoformat_ and supported by your _date.fromisoformat_, as far as I know.